### PR TITLE
Coinjoin timers update

### LINKED ISSUE #6627: Next CoinJoin round counter
Improve the precision of countdown counter for the next CoinJoin round to `seconds` .

### LINKED ISSUE #6746: Add en estimate for Coinjoin session end
Display an estimate for the coinjoin session end in the coinjoin summary section. Use the `getEstimatedTimePerRound()` util for calculation.

### PR DESCRIPTION
**Pull request by dahaca**
## Description

- **feat:** display phase countdowns with seconds everywhere
- **feat:** calculate an estimated session deadline
- **chore:** removed the anonymity chart

## Related Issue

Resolve [#6627](https://github.com/trezor/trezor-suite/issues/6627) #6746

### CODE REVIEW

File: packages/suite/src/utils/wallet/__tests__/coinjoinUtils.test.ts [hunk 1]
```diff
@@ -60,3 +60,33 @@ describe('breakdownCoinjoinBalance', () => {
         });
     });
 });
+
+describe('getSessionDedlineFormat', () => {
+    global.Date.now = jest.fn(() => new Date('2022-02-22T22:22:22Z').getTime());
```
Comment (szymonlesisz) [inline packages/suite/src/utils/wallet/__tests__/coinjoinUtils.test.ts]:
```suggestion
    jest.spyOn(Date, 'now').mockImplementation(() => new Date('2022-02-22T22:22:22Z').getTime());
```

File: packages/suite/src/reducers/wallet/coinjoinReducer.ts [hunk 2]
```diff
@@ -85,13 +89,31 @@ const updateSession = (
     const account = draft.accounts.find(a => a.key === accountKey);
     if (!account || !account.session) return;
 
+    const { signedRounds, maxRounds, skipRounds } = account.session;
+    const { phase, phaseDeadline, roundDeadline } = round;
+
+    let sessionDeadline: string | number;
+    const roundsLeft = maxRounds - signedRounds?.length;
```
Comment (szymonlesisz) [inline packages/suite/src/reducers/wallet/coinjoinReducer.ts]:
```suggestion
    const roundsLeft = maxRounds - signedRounds.length - (typeof phase === 'number' ? 1 : 0)
```
- signedRounds is not optional, should be defined here
- exclude current round (if registered).  you know current round estimation from `roundDeadline` so you don't have to calculate it in getEstimatedTimePerRound
- sum sessionDeadlineRaw + timeLeftTillRoundEnd

**Review by szymonlesisz — CHANGES_REQUESTED** [review]

File: packages/suite/src/components/wallet/CoinjoinSummary/CoinjoinStatus.tsx [hunk 3]
```diff
@@ -130,18 +132,16 @@ export const CoinjoinStatus = ({
...
     const theme = useTheme();
 
-    const timeLeft = `${
-        (session.maxRounds - session.signedRounds.length) *
-        getEstimatedTimePerRound(!!session.skipRounds)
-    }h`;
-    const progress = session.signedRounds.length / (session.maxRounds / 100);
+    const { maxRounds, signedRounds, paused, sessionDeadline, phase } = session;
+
+    const progress = signedRounds.length / (maxRounds / 100);
```
Comment (mroz22) [inline packages/suite/src/components/wallet/CoinjoinSummary/CoinjoinStatus.tsx]: candidate for shared util. also can be found here packages/suite/src/components/suite/CoinjoinStatusBar.tsx

File: packages/suite/src/utils/wallet/__tests__/coinjoinUtils.test.ts (hunk 1 above)
Comment (dahaca) [reply packages/suite/src/utils/wallet/__tests__/coinjoinUtils.test.ts]: done

File: packages/suite/src/reducers/wallet/coinjoinReducer.ts (hunk 2 above)
Comment (dahaca) [reply packages/suite/src/reducers/wallet/coinjoinReducer.ts]: done

File: packages/suite/src/components/wallet/CoinjoinSummary/CoinjoinStatus.tsx (hunk 3 above)
Comment (dahaca) [reply packages/suite/src/components/wallet/CoinjoinSummary/CoinjoinStatus.tsx]: done

**Comment by dahaca** [comment]
@matejkriz @hynek-jina I had to leave full words for time (`minutes` not `min`) because I think we would need custom `Locales` for `formatDuration()` with short forms, translated to all languages :( A short version would be looking much better for sure.

**Review by szymonlesisz — APPROVED** [review]
ack on coinjoinReducer changes i didnt test it tho

File: packages/suite/src/utils/wallet/__tests__/coinjoinUtils.test.ts [hunk 4]
```diff
@@ -60,3 +60,31 @@ describe('breakdownCoinjoinBalance', () => {
         });
     });
 });
+
+describe('getSessionDedlineFormat', () => {
+    jest.spyOn(Date, 'now').mockImplementation(() => new Date('2022-02-22T22:22:22Z').getTime());
+
+    it('works with hours, munites and seconds left', () => {
```
Comment (matejkriz) [inline packages/suite/src/utils/wallet/__tests__/coinjoinUtils.test.ts:67]: typo "munites" here and on line 75

File: packages/suite/src/utils/wallet/coinjoinUtils.ts [hunk 5]
```diff
@@ -226,3 +226,38 @@ export const prepareCoinjoinTransaction = (
         },
     };
 };
+
+export const getSessionDedlineFormat = (deadline: CoinjoinSession['sessionDeadline']) => {
```
Comment (matejkriz) [inline packages/suite/src/utils/wallet/coinjoinUtils.ts]: typo
```suggestion
export const getSessionDeadlineFormat = (deadline: CoinjoinSession['sessionDeadline']) => {
```

**Comment by hynek-jina** [comment]
@dahaca 
I think it's not a problem to translate short versions too..

**Pull request merged**